app/components/learner-group/bulk-assignment.js

```diff
@@ -11,6 +13,8 @@ export default class LearnerGroupBulkAssignmentComponent extends Component {
   @tracked validUsers = [];
   @tracked matchedGroups = [];
 
+  @use allDescendants = new ResolveAsyncValue(() => [this.args.learnerGroup.allDescendants, []]);
```

Comment:
`learnerGroup.allDescendants` isn't async it's a getter on the model and can be access directly as it will return an array when it's empty.